.github/workflows/open-v3-maintenance-prs.yml

```diff
@@ -52,6 +52,10 @@ jobs:
 
             Depending on your changes, running `git rebase --onto v3-maintenance main ${{ github.head_ref }}` might be a good starting point.
 
+            Notes:
+            - your PR branch should be named `v3-backport-${{ github.event.number }}`
```

Comment:
This isn't required, it's just what the automatic workflow names it

Reply:
But it is useful to have here in case you need to create your own backport PR

Reply:
I think it is when CI checks if the v4 PR was merged.